### DISCUSSION TITLE
Cant find Addon in Home Assistant

### DISCUSSION THREAD

**Discussion by Sentinel-sys** [mak-gitdev/HA_enoceanmqtt#68] [Q&A]
Hi there,
just wanted to install the Addon in Home Assistant. But when I add the repository https://github.com/mak-gitdev/HA_enoceanmqtt-addon there are only two Addons:
EnOcean MQTT (******* DEPRECATED *******)
EnOcean MQTT (dev)
Is there no "regular" one?
I also installed the EnOcean MQTT (dev) but I was unable to connect to my USB300 USB Stick.
Thank you
Joe

**Comment by cubertt** [comment]
i am using the (dev) version, seems to be the latest version.

**Comment by chma73** [comment]
Hello, I have maybe a similar problem, when I add the repository https://github.com/mak-gitdev/HA_enoceanmqtt-addon I get no add-on to install. Can someone support me how I can solve it?
Thanks
Chris

**Reply by chma73** [reply]
ah now it's shown - solved :)

**Comment by mak-gitdev** [comment]
Hi @Sentinel-sys,

Sorry I missed your question.
Did you solve your issue ?
As @cubertt said, only the dev version is available at the moment.

**Comment by remy3885** [comment]
Hello, when I try to install the dev version I get an error. It is impossible to install it. Do you have any idea of ​​the problem?

> Échec de l'installation du module complémentaire
The command '/bin/ash -o pipefail -c apk add --no-cache python3 py3-pip git && python3 -m venv /app/venv && . /app/venv/bin/activate && python3 -m pip install --upgrade pip && pip3 install pyyaml && pip3 install tinydb && pip3 install paho-mqtt==1.6.1 && pip3 install git+https://github.com/mak-gitdev/enocean.git && git clone -b master --single-branch --depth 1 https://github.com/embyt/enocean-mqtt.git && cd enocean-mqtt && python3 setup.py develop && cd .. && if [ "${BUILD_VERSION}" = "head" ]; then echo "Installing from repository head" && git clone -b develop --single-branch --depth 1 https://github.com/mak-gitdev/HA_enoceanmqtt.git ; else if [ -z "${BUILD_VERSION##*.*.*}" ]; then echo "Installing development version ${BUILD_VERSION%-*}" ; else echo "Installing stable version ${BUILD_VERSION%-*}" ; fi; wget -nv -O "/app.tar.gz" "https://github.com/mak-gitdev/HA_enoceanmqtt/archive/refs/tags/${BUILD_VERSION%-*}.tar.gz" && echo "Extracting /app.tar.gz" && tar xzf "/app.tar.gz" && rm "/app.tar.gz" && mv -v "HA_enoceanmqtt-${BUILD_VERSION%-*}" HA_enoceanmqtt ; fi; cp -rf HA_enoceanmqtt/enoceanmqtt enocean-mqtt && rm -rf HA_enoceanmqtt' returned a non-zero code: 1

**Reply by ChristopheHD** [reply]
Issue already raised : https://github.com/mak-gitdev/HA_enoceanmqtt/issues/161
